lib/Driver/ToolChains.cpp

```diff
@@ -1690,7 +1954,7 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
     Arguments.push_back("-lswiftCore");
   }
   
-  // Explicitly pass the target to the linker
+  // Explicitly pass the target to the driver
```

Comment:
"the driver" means "the Swift driver" in this context. If you don't like "the linker", please just say "Clang" here.